source/index.html

```diff
@@ -96,47 +97,41 @@ <h3 class="advantages__title">мало туристов</h3>
           <span class="advantages__number">— №5 —</span>
           <p class="advantages__text">большинство едет в гранд каньон и толпится там</p>
         </li>
-      </ul>
+      </ol>
     </section>
     <section class="interested">
       <h2 class="interested__title">Заинтересовались?</h2>
-      <p class="interested__text">Укажите предполагаемые даты поездки, и мы покажем вам лучшие предложения гостиниц в седоне</p>
-      <button class="search-button" type="button">Найти гостиницу</button>
+      <p class="interested__text">Укажите предполагаемые даты поездки, и мы покажем вам лучшие предложения гостиниц в
+        седоне</p>
+      <a class="search-button button" href="#">Найти гостиницу</a>
```

Comment:
Будет логично смиксовать общий БЭМ-блок `button` не с другим блоком, а с элементом родительского блока, примерно так:
```suggestion
      <a class="interested__button button" href="#">Найти гостиницу</a>
```